src/components/pages/career-details/career-details-structured-data.tsx

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import { Helmet } from 'react-helmet';
+import { SyPersonioJob } from '../../../types';
+
+interface CareerDetailsStructuredDataProps {
+  position: SyPersonioJob;
+}
+
+const convertScheduleToEmploymentType = (schedule: string): string => {
```

Comment:
Please comment on why this is necessary. Maybe it's not even necessary to put it into its own method, but just use it where it is needed and add the comment above.

Reply:
I provided a comments on behalf of @milljoniaer  🙏